src/sgame/sg_bot.cpp

```diff
@@ -727,10 +737,35 @@ void G_BotFill(bool immediately)
 		}
 	}
 
+	int biggerTeamSize = -1;
+	for ( team_t team : { TEAM_ALIENS, TEAM_HUMANS } )
```

Comment:
I don't understand what the calculation in this loop is supposed to be doing. If you want to get the size of a team excluding filler bots, you can subtract the length of the filler bots list from the team size. I think the body of the loop could just be
```
if ( <relevant cvar>.Get() )
   fillers[ team ].actualFill = std::max( fillers[ team ].actualFill, level.team[ team ].numClients - fillers[ team ].current.size();